specification/dartLangSpec.tex

```diff
@@ -16611,7 +16623,7 @@ \subsection{Identifier Reference}
 \LMHash{}%
 It is a compile-time error if either of the identifiers \AWAIT{} or \YIELD{}
 is used as an identifier in a function body
```

Comment:
Here we say "as an identifier" too. Should we elaborate that ("as a production of \<identifier\>")?

Reply:
Done!